addons/mrp/static/tests/mock_server/mock_models/mrp_document.js

```diff
@@ -0,0 +1,5 @@
+import { ServerModel } from "@web/../tests/_framework/mock_server/mock_model";
```

Comment:
I think we'd better importing it from `@web/../tests/web_test_helpers` as it seems to be the public entry point.